### DISCUSSION TITLE
WARNING: linker : /data/data/com.termux/files/usr/bin/sh: unused DT entry

### DISCUSSION THREAD

**Discussion by oktaysavci** [termux/termux-app#3534] [Q&A] [answered]
Hello everyone. I'm getting such an error in Termux's version (0.119.1). Help please!

WARNING: linker : /data/data/com.termux/files/usr/bin/sh: unused DT entry: type 0x1d arg 0x2cb CANNOT LINK EXECUTABLE: cannot locate symbol "strc hrnul" referenced by "/data/data/com.termux/files/ usr/bin/sh"...

[Process completed (code 1) - press Enter][](url)

**Comment by oktaysavci** [comment]
My Phone: Sony Xperia E5 (F3311)
Android version: 6.0
Ram: 1.5GB

**Comment by sylirre** [comment, marked as answer]
Builds made for Android 7 would not work on older Android versions. Please choose the correct builds from GitHub Actions. They are labeled "android-5".

**Reply by oktaysavci** [reply]
I am using Android 6.0. Is there a version of Termux that works for Android 6.0?

**Reply by sylirre** [reply]
You need to use APK built for Android 5: https://github.com/termux/termux-app/wiki/Termux-on-android-5-or-6

Open the latest GitHub Actions run on the branch `master` you can find the list of artifacts. Some are with `android-7` label and others are with `android-5`. Use one with `android-5`. Make sure correct arch selected, otherwise use universal build.

Here is a link to universal build compatible with Android 5 & 6: https://github.com/termux/termux-app/suites/13849110837/artifacts/769225147

**Reply by oktaysavci** [reply]
Thank you.
